Package/PackageActions/Install.cs

```diff
@@ -190,7 +190,7 @@ private int DoExecute(CancellationToken cancellationToken)
 
                 RaiseProgressUpdate(10, "Gathering dependencies.");
                 bool checkDependencies = (!IgnoreDependencies && !Force) || CheckOnly;
-                var issue = DependencyChecker.CheckDependencies(installationPackages, packagesToInstall,
+                var issue = DependencyChecker.CheckDependencies(installationPackages.Where(p => p.IsSystemWide() == false), packagesToInstall,
```

Comment:
What happens if an installed package requires a system-wide package to be installed? Does those dependencies then fail?

Also, it'd be good with a comment about why we are removing system wide packages from the list of dependencies.

Reply:
I understand your point. Maybe we need to discuss the optimal behavior here.

Reply:
After testing i can confirm that you still get a system-wide package (PathWave License Manager) installed as a dependency e.g. with `tap package install "Switch Manager"`